TheraLang.BLL/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using TheraLang.BLL.DataTransferObjects;
+using TheraLang.BLL.Interfaces;
+using TheraLang.DAL.Entities;
+using TheraLang.DAL.UnitOfWork;
+
+namespace TheraLang.BLL.Services
+{
+    public class AuthenticationService : IAuthenticateService
+    {
+        private readonly TokenManagement _tokenManagement;
+        private readonly IHttpContextAccessor _context;
+
+        public AuthenticationService(IOptions<TokenManagement> tokenManagement, IHttpContextAccessor context)
+        {
+            _tokenManagement = tokenManagement.Value;
+            _context = context;
+        }
+
+        public async Task<string> Authenticate(User user)
```

Comment:
You removed "async" from method name, and this method doesn't contains await, so method is not async, so why you left async operator in method signature?